hack/run-ci-e2e-test.sh

```diff
@@ -108,6 +108,9 @@ OSDK_WMCO_test $OSDK "-run=TestWMCO/operator_deployed_without_private_key_secret
 # Run the creation tests and skip deletion of the Windows VMs
 OSDK_WMCO_test $OSDK "-run=TestWMCO/create -v -timeout=90m -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION"
 
+# Run the upgrade tests and skip deletion of the Windows VMs
+OSDK_WMCO_test $OSDK "-run=TestWMCO/upgrade -v -timeout=90m -node-count=$NODE_COUNT -skip-node-deletion -ssh-key-pair=$KEY_PAIR_NAME --private-key-path=$KUBE_SSH_KEY_PATH $WMCO_PATH_OPTION"
```

Comment:
-skip-node-deletion is not needed. since you specified `-run=...upgrade` the deletion tests will not run

Reply:
Removing this from upgrade here. Creating a separate PR to remove the flag itself.

Reply:
@VaishnaviHire yes I too think the `-skip-node-deletion` flag is not needed in general. So removing it in a separate PR is fine. However you don't need to add it to this line if it is not needed.